Pandoc interface has changed

```
pandoc-plantuml-diagrams-0.1.0.3/src/Text/Pandoc/PlantUML/Filter/OutputBlock.hs:22:60: error:
    • Couldn't match expected type ‘[Inline]’
                  with actual type ‘(String, [Char])’
    • In the second argument of ‘Image’, namely
        ‘((show imageFileName), "fig:")’
      In the expression:
        Image (altTagInline attr) ((show imageFileName), "fig:")
      In an equation for ‘imageTag’:
          imageTag imageFileName attr
            = Image (altTagInline attr) ((show imageFileName), "fig:")
   |
22 | imageTag imageFileName attr    = Image (altTagInline attr) ((show imageFileName), "fig:")
   |                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,29 @@
-language: haskell
+dist: xenial
+
+language: generic
+
+cache:
+  directories:
+    - $HOME/.stack
+
+addons:
+  apt:
+    packages:
+      - libgmp-dev
+
+before_install:
+# Download and unpack the stack executable
+- mkdir -p ~/.local/bin
+- export PATH=$HOME/.local/bin:$PATH
+- travis_retry curl -L https://get.haskellstack.org/stable/linux-x86_64.tar.gz | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
+
+install:
+# Build dependencies
+- stack --no-terminal --install-ghc test --only-dependencies
+
+script:
+# Build the package, its tests, and its docs and run the tests
+- stack --no-terminal test --haddock --no-haddock-deps
 
 deploy:
   provider: hackage

--- a/pandoc-plantuml-diagrams.cabal
+++ b/pandoc-plantuml-diagrams.cabal
@@ -1,5 +1,5 @@
 name:                pandoc-plantuml-diagrams
-version:             0.1.0.3
+version:             0.1.0.4
 synopsis:            Render and insert PlantUML diagrams with Pandoc
 description:         PlantUML renders different types of UML diagrams.
                      This filter invokes plantuml.jar (which must be present
@@ -47,6 +47,17 @@ Test-Suite test-pandoc-crossref
   Main-Is:             Spec.hs
   hs-source-dirs:      test, src
   default-language:    Haskell2010
+  other-modules:       Text.Pandoc.PlantUML.Filter
+                     , Text.Pandoc.PlantUML.Filter.IORender
+                     , Text.Pandoc.PlantUML.Filter.Types
+                     , Text.Pandoc.PlantUML.Filter.FileNameGenerator
+                     , Text.Pandoc.PlantUML.Filter.Formats
+                     , Text.Pandoc.PlantUML.Filter.OutputBlock
+                     , Text.Pandoc.PlantUML.Filter.FileNameGeneratorSpec
+                     , Text.Pandoc.PlantUML.Filter.FormatsSpec
+                     , Text.Pandoc.PlantUML.Filter.OutputBlockSpec
+                     , Text.Pandoc.PlantUML.Filter.TypesSpec
+                     , Text.Pandoc.PlantUML.FilterSpec
   Build-depends:       hspec
                      , hspec-discover
                      , base >= 4 && < 5
@@ -54,6 +65,9 @@ Test-Suite test-pandoc-crossref
                      , pandoc-types
                      , SHA
                      , utf8-string
+                     , bytestring
+                     , process
+                     , directory
 
 source-repository head
   type:                git

--- a/src/Text/Pandoc/PlantUML/Filter/IORender.hs
+++ b/src/Text/Pandoc/PlantUML/Filter/IORender.hs
@@ -1,4 +1,4 @@
-
+{-# LANGUAGE BangPatterns #-}
 -- | Module: Text.Pandoc.PlantUML.Filter.Render
 -- Defines the actual rendering done with PlantUML
 module Text.Pandoc.PlantUML.Filter.IORender() where
@@ -26,6 +26,6 @@ plantUmlProcess (ImageFileName _ fileType) = (proc "java" ["-jar", "plantuml.jar
 
 pipe :: Handle -> Handle -> IO ()
 pipe hIn hOut = do
-  input <- hGetContents hIn
+  !input <- hGetContents hIn
   hPut hOut input
 

--- a/src/Text/Pandoc/PlantUML/Filter/OutputBlock.hs
+++ b/src/Text/Pandoc/PlantUML/Filter/OutputBlock.hs
@@ -19,7 +19,7 @@ resultBlock :: ImageFileName -> Attr -> Block
 resultBlock imageFileName attr = Para $ map (\p -> p imageFileName attr) [imageTag, idTag]
 
 imageTag :: ImageFileName -> Attr -> Inline
-imageTag imageFileName attr    = Image nullAttr (altTagInline attr) ((show imageFileName), "fig:")
+imageTag imageFileName attr    = Image nullAttr (altTagInline attr) (show imageFileName, "fig:")
 
 idTag :: ImageFileName -> Attr -> Inline
 idTag _ (id, _, _)             = Str ("{#" ++ id ++ "}")

--- a/stack.yaml
+++ b/stack.yaml
@@ -1,0 +1,1 @@
+resolver: lts-14.15

